docs/Getting-Started-CUDA.md

```diff
@@ -1,31 +1,44 @@
 # Getting Started with CUDA
 
-**Note** that right now CUDA support is very preliminary and Levanter does not work on a multi-machine setup.
+**Note** that right now CUDA support is very preliminary and Levanter does not work on a multi-machine setup. (We think this is just a dependency/environment issue.)
 
-## Setting up dependencies
+We only test on Ampere (e.g. A100s or 30xx series)  GPUs. If it works with Jax it should work, though.
+
+## Installation
+### Setting up Conda Environment
 ```bash
 # 3.11 is too new for tensorstore
 # this doesn't actually install pytorch, but it bundles cuda dependencies nicely
 conda create --name levanter -c pytorch -c nvidia pytorch-cuda=11.7 python~=3.10
 conda activate levanter
-pip install "jax[cuda11_cudnn82]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-pip install -r requirements.txt
+```
+### Install Jax with CUDA
+
+This is a bit in flux in Jax, but please take a look at the [Jax Installation Guide](https://github.com/google/jax#pip-installation-gpu-cuda-installed-via-pip-easier). Here are two options that work circa June 2023:
+
+```bash
+# CUDA 12 installation
+pip install --upgrade "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+
+# CUDA 11 installation
+pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
 ```
 
+### Install Levanter
+
 ```bash
-# optional: install go to get gpu heap utilization
-# nvidia-smi doesn't work when using Jax since it preallocates all the memory
-sudo apt-get install golang
-# or do this if you don't have sudo:
-conda install -c conda-forge go
+cd levanter
+pip install -e .
 ```
 
 ## Running a job on Slurm
 
 ### Single Node
 
+Simple example, customize to your needs:
+
 ```bash
-srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx[7-8] --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python examples/gpt2_example.py --config_path config/gpt2_small.yaml
+srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx7 --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python examples/gpt2_example.py --config_path config/gpt2_small.yaml
```

Comment:
Note that I just tested in sphin6, which used CUDA 11.7. I installed jax 0.4.11. 

I had this issue when running Levanter. This is consistent with what I encountered on jagupard (P6000):

```
FAILED tests/test_backpack.py::test_backpack_predict - jaxlib.xla_extension.XlaRuntimeError: INTERNAL: nvlink exited with non-zero error code 256, output: nvlink fatal   : Input file '/tmp/ivanzhou/tempfile-sphinx6-efa86375-3475856-5fd80f1d5370f.cubin' newer than toolkit (118 vs 113)
```

Reply:
I don't think this is an issue of Levanter. This is more likely something between CUDA toolkit and Jax.

Reply:
<img width="1133" alt="image" src="https://github.com/stanford-crfm/levanter/assets/9633/6cbaf773-2f50-44b9-a5cd-bcccacecda6a">

Yeah looks like we need to increase the support version range and/or require people to build form source on sphinx and/or get the sphinxes upgraded. We should probably do all three.

Reply:
(that's a screenshot of the current jax readme)